### ISSUE TITLE
Singapore Channel

### ISSUE THREAD

**Issue by beisongc** [iptv-org/iptv#1697]
**Country of the channel:**
Singapore

**Channel Name:**
Channel 8, Channel 5, Channel U

**Channel Language:**
Channel 8: Chinese
Channel 5: English
Channel U: Chinese

**Additional Information (optional):**
Able to watch from any IP or VPN SG.

**Comment by bertperrisor** [comment]
@beisongc @freearhey 
I think everyone can say goodbye to grab the streams from Mewatch 😁. They are protecting it with AES encryption for DRM now.
```
$ curl -s https://tglmp02.akamaized.net/out/v1/f091238359f8455f8396cef639d0d9f7/master_1.m3u8
#EXTM3U
#EXT-X-VERSION:5
#EXT-X-TARGETDURATION:4
#EXT-X-MEDIA-SEQUENCE:11609748
#EXT-X-KEY:METHOD=SAMPLE-AES,URI="skd://entry-fta_ch5_hls_web_drm",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1",IV=0x5A5926669F76F04799D18E106212745F
...
```
I can verify this in Kodi logs.
```
2020-08-24 22:55:57.825 T:123145399209984    INFO: ffmpeg[700005C69000]: [https] Opening 'https://tglmp02.akamaized.net/out/v1/f091238359f8455f8396cef639d0d9f7/master_1.m3u8' for reading
2020-08-24 22:55:57.909 T:123145399209984   ERROR: ffmpeg[700005C69000]: [hls,applehttp] SAMPLE-AES encryption is not supported yet
2020-08-24 22:55:57.909 T:123145399209984   DEBUG: ffmpeg[700005C69000]: [hls,applehttp] Failed to open segment 11609693 of playlist 0
2020-08-24 22:55:57.909 T:123145399209984   ERROR: ffmpeg[700005C69000]: [hls,applehttp] SAMPLE-AES encryption is not supported yet
2020-08-24 22:55:57.909 T:123145399209984   DEBUG: ffmpeg[700005C69000]: [hls,applehttp] Failed to open segment 11609694 of playlist 0
...
```

**Comment by beisongc** [comment]
@bertperrisor 
thank you very much for your help.

**Comment by exodiver** [comment]
That sux man.

**Comment by freearhey** [comment]
Closed in favor #1983

**Comment by teamelite2019** [comment]
looking for channel 8, anyone can help ?
